pkg/kubelet/kuberuntime/kuberuntime_container_windows.go

```diff
@@ -125,9 +124,31 @@ func (m *kubeGenericRuntimeManager) generateWindowsContainerConfig(container *v1
 
 	// setup security context
 	effectiveSc := securitycontext.DetermineEffectiveSecurityContext(pod, container)
+
+	// Strip down all the unnecessary options on the Windows
```

Comment:
Is it worth adding a comment around what is available vs what is not for future contributors.  I could see it easily coming to this and wondering why some fields are cleared and some are not

Reply:
Setting the fields of `effectiveSc` to `nil` in this function is not necessary as [discussed](https://github.com/kubernetes/kubernetes/pull/93475/files#r463014435) - those fields aren't used/referred to anywhere right now. It also has adds a little bit of a maintenance burden as @jsturtevant mentions above. The core intent here is to avoid returning the error [below](https://github.com/kubernetes/kubernetes/pull/93475/files#diff-4d8e81d13b02ca9a4b8919ef8cf8b6f9L130). So how about just removing the error? That error was introduced in https://github.com/kubernetes/kubernetes/pull/64009 before `WindowsOptions.RunAsUserName` was introduced in https://github.com/kubernetes/kubernetes/pull/79489 and documented as the way to pass a username for containers. So removing just that error should be good now (rather than setting the various fields to nil). Thoughts, @jsturtevant ?

Reply:
It seems that https://github.com/kubernetes/kubernetes/pull/93475/files#diff-0170c1534446e377b025a1c54ab478e2R176 also effectively doesn't copy the RunAsUserName into the Security context.  Removing the dependency with #93580 should make this even better.